src/core/unit_tests/Array_test.cpp

```diff
@@ -79,3 +82,27 @@ BOOST_AUTO_TEST_CASE(broadcast) {
   static_assert(a[1] == 5, "");
   static_assert(a[2] == 5, "");
 }
+
+BOOST_AUTO_TEST_CASE(serialization) {
```

Comment:
Why do you use MPI here?

Reply:
To test serialization... of course I could also just serialize and deserialize it